sdktests/server_side_stream_retry.go

```diff
@@ -0,0 +1,256 @@
+package sdktests
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/launchdarkly/sdk-test-harness/framework/harness"
+	"github.com/launchdarkly/sdk-test-harness/framework/ldtest"
+	"github.com/launchdarkly/sdk-test-harness/mockld"
+	"github.com/launchdarkly/sdk-test-harness/servicedef"
+
+	"github.com/launchdarkly/go-test-helpers/v2/httphelpers"
+	m "github.com/launchdarkly/go-test-helpers/v2/matchers"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/ldtime"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
+)
+
+const briefDelay ldtime.UnixMillisecondTime = 1
+
+func baseStreamConfig(endpoint *harness.MockEndpoint) *servicedef.SDKConfigStreamingParams {
+	return &servicedef.SDKConfigStreamingParams{
+		BaseURI:             endpoint.BaseURL(),
+		InitialRetryDelayMs: timeValueAsPointer(briefDelay),
+	}
+}
+
+func doServerSideStreamRetryTests(t *ldtest.T) {
```

Comment:
As with the update tests, these are implemented pretty tediously in each SDK's unit tests and we've often forgotten some edge cases there. It's impossible for integration-harness to test any of these scenarios.